app/views/users/show.html.erb

```diff
@@ -1,25 +1,28 @@
-<h1>Here is the page for a specific user</h1>
-<div class="user-repeat">
-  <%= render partial: "shared/username", locals: { user: @user } %>
+<h2 class="topic">Specific user</h2>
+<%= render partial: "shared/username", locals: { user: @user } %>
+
+<div class="card">
+  <div class="card-body">
+    <h5 class="card-title">Bio</h5>
+    <p class="card-text"><%= @user.bio %></p>
+  </div>
 </div>
 
-<div class="user-id-bio">
-  <p>Bio</p>
-  <p> <%= @user.bio %>
+<div class="card">
+    <% @user.recent_post.each do |post| %>
+      <div class="card-body">
+        <h5 class="card-title"><%= post.title %></h5>
+        <h6 class="card-subtitle mb-2 text-muted"><%= post.text %></h6>
+        <p class="card-text">Comments: <small class="text-muted"><%= post.comments_counter%></small>, Likes: <small class="text-muted"><%= post.likes_counter%></small></p>
+      </div>
+    <% end %>
 </div>
 
-<div>
-  <% @user.recent_post.each do |post| %>
-    <h4><%= link_to post.title, user_post_path(post) %></h4>
-    <p><%= post.text %></p>
-    <div>
-      <span>Comments: <%= post.comments_counter%>,</span> 
-      <span>Likes: <%= post.likes_counter%></span>
-    </div>
-  <% end %>
+<div class="centro-user-show-button">
+  <a>
+    <%= link_to "See all posts", user_posts_path(@user), :class => "btn btn-primary" %>
+  </a>
```

Comment:
- [ ] I suggest that you make this user page looks like the given wireframe
![flame](https://user-images.githubusercontent.com/44745855/150859771-6cdef28e-0529-4209-a2a7-e9701abdbb8f.png)